docs/kit/hardware/brain-board.md

```diff
@@ -0,0 +1,44 @@
+# Brain Board (Raspberry Pi)
+
+![Raspberry Pi 4](https://upload.wikimedia.org/wikipedia/commons/f/f1/Raspberry_Pi_4_Model_B_-_Side.jpg)
+
+There are two LEDs on the Raspberry Pi itself: `PWR` and `ACT`. Both are located on the lower left side of the board.
+
+The LEDs on the KCH can also provide indication as to the state of the Raspberry Pi.
+
+### Power LED (red)
+
+The red `PWR` LED broadly indicates that there is 5V power to the Raspberry Pi.
+
+Unlike other RPI, on a RPI4 the `PWR` LED is fully under the control of a GPIO expander, and when booting this IO expander is reset, causing the `PWR` LED to blink off on reboot. On booting the bootloader enables it again.
+
+But if the PWR LED goes off (blinks) at any other time it means have an unfit power supply/power cable. 
+
+In short, the PWR LED should be always on except for a very short time just before a reboot happens.
+
+### Activity LED (Green)
+
+The green `ACT` LED will flash "randomly" during normal usage and indicates access to the SD card.
+
+If the Pi fails to boot, the `ACT` LED should flash in a pattern, listed below. If the `ACT` LED does not turn on, even though the `PWR` LED does, this indicates that the EEPROM is corrupt. It may be fixed using the `recovery.bin` method, which can be performed using [Raspberry Pi Imager](https://www.raspberrypi.com/software/).
+
+| Long Flashes | Short Flashes | Status                        | Probable reason                       |
```

Comment:
What's the source of this table -- is this something in our software, or is this copied from official Raspberry Pi docs? If the latter it would probably make sense to include a link to the original, even if we retain the copy for convenience.

Reply:
I've collated this from various sources around the internet. The source for the status is mostly from forum posts by raspberry pi staff. The probable reasons and recommendations have been written by myself.

Reply:
Ah, interesting. That might be worth noting inline?

Reply:
13d167c